Gemfile

```diff
@@ -43,3 +43,8 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
```

Comment:
You should also use the `jbuilder` gem. `jbuilder` allows you to simplify the rendering of `json` responses

Reply:
It's just "render json: {variable}" now. Feels simple enough. Do I really need this gem?